app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
if you don't need it - remove. Don't be afraid, because you can create it when you need it in one command